### PR TITLE
Minor update in docs w.r.t the term "knowledge graph"

### PR DESCRIPTION
**Pull request by lolski**
# Why is this PR needed?
We need to update terms in the documentations

# What does the PR do?
Updating the following terms:
1. KBMS -> KGMS
2. Knowledge Base -> Knowledge Graph (capitals)
3. knowledge base -> knowledge graph (small case)
4. hyper-relational database -> knowledge graph

# Does it break backwards compatibility?
No

# List of future improvements not on this PR
N/A

### CODE REVIEW

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Introduction to Grakn
 keywords: getting started
 last_updated: September 2017
-summary: This is a brief introduction to what a knowledge based system is, what is GRAKN and the broad topics that will be covered into the Grakn Academy.
+summary: This is a brief introduction to what a knowledge graphd system is, what is GRAKN and the broad topics that will be covered into the Grakn Academy.
```
Comment (kasper-piskorski) [inline docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: :goat:

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md [hunk 2]
```diff
@@ -26,33 +26,33 @@ Let us tart with the most basic question.
 
 In simple and cheesy terms, GRAKN is a database on steroids that makes it easy to build intelligent applications.
 
-More precisely, it is what we call a [Knowledge Based System](https://en.wikipedia.org/wiki/Knowledge-based_systems): a piece of software that allows to model complex problems, made of several components:
+More precisely, it is what we call a [Knowledge Graphd System](https://en.wikipedia.org/wiki/Knowledge-based_systems): a piece of software that allows to model complex problems, made of several components:
```
Comment (kasper-piskorski) [inline docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: :goat:

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md [hunk 3]
```diff
@@ -61,9 +61,9 @@ In this section you will find questions and exercises that you should be able to
 
 For this lesson, the exercises are quite free-form, so feel free to answer them as you see fit:
 
-  * What are the main components of a knowledge based systems?
+  * What are the main components of a knowledge graphd systems?
```
Comment (kasper-piskorski) [inline docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: :goat:

File: docs/pages/academy/6-analytics/3-graph-analytics.md [hunk 4]
```diff
@@ -13,18 +13,18 @@ KB: academy
...
+Well the answer is that an entity-relationship conceptual model (thus a GRAKN knowledge graph) can actually be represented in the form of a graph.
 
 And if you have a graph, then it is often useful to be able to do perform graph analytics.
 
 
 ## A few words before going on
 As you will see, it is quite straightforward to perform graph analytics with Grakn. There are two caveats though:
 
-  1. GRAKN is not a graph database, so to forse a knowledge based with a underlying hypergraph structure into a graph in order to perform graph analytics requires some getting used to.
+  1. GRAKN is not a graph database, so to forse a knowledge graphd with a underlying hypergraph structure into a graph in order to perform graph analytics requires some getting used to.
```
Comment (kasper-piskorski) [inline docs/pages/academy/6-analytics/3-graph-analytics.md]: :goat:, also FORSE

File: docs/pages/academy/6-analytics/3-graph-analytics.md [hunk 5]
```diff
@@ -13,18 +13,18 @@ KB: academy
...
   2. Actually using the results of graph analytics queries requires some programming expertise (except for the shortest path). So this is probably the most advanced topic covered so far in the Academy.
 
-To cast GRAKN hyper-relational knowledge bases as a graph, imagine that each concept (i.e. an entity, a relationship or an attribute) is a node in your graph and two nodes are connected via an edge if there is a relationship in which both concept play a role. This is easier to visualise in some cases and requires a bit more imagination in some other, but it is quite natural.
+To cast GRAKN hyper-relational knowledge graphs as a graph, imagine that each concept (i.e. an entity, a relationship or an attribute) is a node in your graph and two nodes are connected via an edge if there is a relationship in which both concept play a role. This is easier to visualise in some cases and requires a bit more imagination in some other, but it is quite natural.
```
Comment (kasper-piskorski) [inline docs/pages/academy/6-analytics/3-graph-analytics.md]: what about hyper-relational knowledge graph compound?

Comment (lolski) [reply docs/pages/academy/6-analytics/3-graph-analytics.md]: I changed the sentence to start from "Imagine that each concept (i.e. ...)" in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f

File: docs/pages/academy/6-analytics/3-graph-analytics.md (hunk 4 above)
Comment (lolski) [reply docs/pages/academy/6-analytics/3-graph-analytics.md]: Hahahahha fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md (hunk 3 above)
Comment (lolski) [reply docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: Fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md (hunk 2 above)
Comment (lolski) [reply docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: Fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f

File: docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md (hunk 1 above)
Comment (lolski) [reply docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md]: Nice catch, fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f

File: docs/pages/docs/0-get-started/grakn-server.md [hunk 6]
```diff
@@ -40,7 +40,7 @@ We recommend changing this to another location to make upgrading Grakn easier an
 
 ### Grakn Cluster
 
-Grakn's clustering functionality is part of [Grakn KBMS](https://grakn.ai/kbms).
+Grakn's clustering functionality is part of [Grakn KGMS](https://grakn.ai/kbms).
```
Comment (duckofyork) [inline docs/pages/docs/0-get-started/grakn-server.md]: maybe https://grakn.ai/grakn-kgms ?

Comment (lolski) [reply docs/pages/docs/0-get-started/grakn-server.md]: fixed in https://github.com/graknlabs/grakn/pull/2681/commits/2c70b33f67c420630da4129ab64f1c26bcf7c58c

**Pull request merged**